velox/exec/RowNumber.cpp

```diff
@@ -440,16 +428,13 @@ void RowNumber::setupInputSpiller(
 
 void RowNumber::spill() {
   VELOX_CHECK(spillEnabled());
-  VELOX_CHECK_NULL(inputSpiller_);
-
-  spillPartitionBits_ = HashBitRange(
-      spillConfig_->startPartitionBit,
-      spillConfig_->startPartitionBit + spillConfig_->joinPartitionBits);
 
   const auto spillPartitionSet = spillHashTable();
-
   setupInputSpiller(spillPartitionSet);
 
+  table_->clear();
```

Comment:
We already did this in spillHashTable()? Can we just check that after spillHashTable(), table_ is cleared?